design/src/high-level/game-thesis.md

```diff
@@ -47,6 +47,7 @@ These are self-imposed, and chosen to create a more compelling and cohesive game
   - Solarpunk, not Zerg
 - Colonies should always feel like bustling organic hives and farms, not mechanized factories.
   - The thematic focus z`should be on "nature reclaiming" instead of "industry civilizing"
```

Comment:
While I like the sentiment here, it's a little difficult to properly understand? There is more of a "thinking/purposeful nature" competing against "idyllic nature" thing here?